src/test/java/rx/subjects/ReplaySubjectConcurrencyTest.java

```diff
@@ -154,7 +154,7 @@ public void onNext(Long args) {
         slowThread.join();
     }
 
-    @Test
+//    @Test
```

Comment:
Btw, there is `@Ignore` annotation, slightly better because you'll see in both Gradle test execution and in the report that there are some ignored tests